combined-update/README.md

```diff
@@ -0,0 +1,41 @@
+# Combined-update-scripts
+
+This repository contains helper scripts for combined update.
+
+## Generating a combined update
+
+`prepare_combined_update.sh` is a helper script for generating a combined update image, later used with FOTA on targets that support combined update. It generates an image with two sub-components: rootfs and bootloader capsule. Where:
+- rootfs image can be either a delta image or a full one.
+- Bootloader capsule consists the u-boot related files that complete the full set required for bootloader update (on platforms that support u-boot capsule update). 
+
+It is assumed that both rootfs and bootloader images are available in an LmP based build platform.
+In addition, the following are required to run the script:
+- sudo access
+- manifest-package-tool/manifest-dev-tool installation 2.4.0 and higher, initialized to work in the current directory
+- mkimage & mkeficapsule utilities
+- Valid package_config.yaml & u-boot-caps.its in the current directory (examples for these are given under the `samples` directory).
+
```

Comment:
Generic note here, I prefer that filenames are given in the `filename` -syntax. Easier to spot that way.